stable/cert-manager/Chart.yaml

```diff
@@ -10,6 +10,11 @@ keywords:
   - tls
 sources:
   - https://github.com/jetstack/cert-manager
-maintainers:
-  - name: munnerz
-    email: james@jetstack.io
+# Deprecated charts cannot have maintainers
```

Comment:
@unguiculus this feels like a weird requirement, but the e2e checker fails if I leave maintainers here 😬